src/Stryker.Core/Stryker.Core/Options/Inputs/AzureFileStorageSasInput.cs

```diff
@@ -0,0 +1,32 @@
+using Stryker.Core.Baseline.Providers;
+using Stryker.Core.Exceptions;
+
+namespace Stryker.Core.Options.Inputs
+{
+    public class AzureFileStorageSasInput : InputDefinition<string>
+    {
+        protected override string Description => "A Shared Access Signature for Azure File Storage is required when azure file storage is used for dashboard compare. For more information: https://docs.microsoft.com/en-us/azure/storage/common/storage-sas-overview";
+
+        public override string Default => string.Empty;
+
+        public string Validate(BaselineProvider baselineProvider)
```

Comment:
It's strange for validation to return a string, I would expect either nothing or boolean.

Reply:
It's actually not only validation but also casting to the desired type. We could change the method name to `CastAndValidate()`?

Reply:
It's not casting either